AN00209_xCORE-200_DSP_Library/app_fft/src/app_fft.xc

```diff
@@ -22,186 +22,222 @@
 #define PRINT_CYCLE_COUNT 1
 #endif
 
-#define INPUT_FREQ N_FFT_POINTS/8
+#define INPUT_FREQ N_FFT_POINTS / 8
 
 #ifndef INT16_BUFFERS
 #define INT16_BUFFERS 0 // disabled by default. int32_t buffers is default
 #endif
 
 #ifndef TWOREALS
-#define TWOREALS 0 // Processing two real signals with a single complex FFT. Disabled by default
+#define TWOREALS \
```

Comment:
We might want to move this comment by hand as clang-format probably can't do better than this on its own.